src/main.rs

```diff
@@ -117,7 +116,10 @@ fn cli_init() -> Result<ValidatedArgs> {
     if let Err(e) = log_cfg.set_time_offset_to_local() {
         eprintln!("WARNING: could not set log TZ to local: {e:?}");
     };
-    log_cfg.set_time_format_rfc3339();
+    log_cfg.set_time_format_custom(format_description!(
+        version = 2,
+        "[year]-[month]-[day]_[hour]:[minute]:[second].[subsecond digits:5]"
```

Comment:
```suggestion
        "[hour]:[minute]:[second].[subsecond digits:4]"
```

I think 4 - a resolution of 100μs - is good enough. But that's not a strongly held opinion.

Also as mentioned by @eugenesvk, shall we remove Y-M-D? I would agree with them that it can be removed but maybe others find value in having it there.

In contrast I'd like to keep `[hour]:` (as we have done here); otherwise there's ambiguity between hour:minute vs. minute:second. Sure it should be obvious to tell anyway, but I still prefer to avoid that moment processing "which one is it actually".

Reply:
The only reason I can think of to keep YMD would be to differentiate between days when saving logs to a file to inspect them later, though I don't know why anyone whould do that.